changelogs/fragments/254-add_ignore_txt.yml

```
@@ -0,0 +1,3 @@
+---
+trivial:
+- Add ignore-2.12.txt (https://github.com/ansible-collections/ansible.netcommon/pull/254).
```

```suggestion
  - Add ignore-2.12.txt (https://github.com/ansible-collections/ansible.netcommon/pull/254).
```

The changelog linter is apparently _real_ strict

---

ok, thanks